.gitpod.yml

```diff
@@ -19,7 +18,7 @@ vscode:
 
 github:
   prebuilds:
-    # enable for master branch
+    # enable for default branch
     master: true
```

Comment:
Interestingly they still maintain `master` as the key here, but [the docs say](https://www.gitpod.io/docs/references/gitpod-yml#prebuildsmaster) it enables prebuilds for the "default" branch, so it looks like we are good?